pkg/services/alerting/notifiers/alertmanager.go

```diff
@@ -0,0 +1,95 @@
+package notifiers
+
+import (
+	"time"
+
+	"github.com/grafana/grafana/pkg/bus"
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/log"
+	m "github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/alerting"
+)
+
+func init() {
+	alerting.RegisterNotifier(&alerting.NotifierPlugin{
+		Type:        "alertmanager",
+		Name:        "alertmanager",
+		Description: "Sends alert to Alertmanager",
+		Factory:     NewAlertmanagerNotifier,
+		OptionsTemplate: `
+      <h3 class="page-heading">Alertmanager settings</h3>
+      <div class="gf-form">
+        <span class="gf-form-label width-10">Url</span>
+        <input type="text" required class="gf-form-input max-width-26" ng-model="ctrl.model.settings.url" placeholder="http://localhost:9093"></input>
+      </div>
```

Comment:
maybe we could add a static list of tags here (if not too hard)

Reply:
I'm not sure it's a good idea to add tags for all alerts using the notifier.
I think we would mess with users, that won't find easily were these tags are defined, and why.

Reply:
The idea was to get a default notified for "page", "alerts" and "tickets". This way it's easy for users to select the severity (they just select the notifier).

This way you can also just configure a notifier instead of putting your tags in each alert (if you want to)

Reply:
Ok, I agree. But, as discussed, we will add this later.